example/src/release_tests/touchables/index.tsx

```diff
@@ -297,7 +301,10 @@ const TOUCHABLES: Touchables[] = [
 ];
 
 const screens: Record<string, Touchables> = TOUCHABLES.reduce(
-  (map: Record<string, Touchables>, obj) => ((map[obj.text] = obj), map),
+  (map: Record<string, Touchables>, obj: Touchables) => {
+    map[obj.text] = obj;
+    return map;
+  },
```

Comment:
Doesn't it do the same thing? If so, why do we change that?

Reply:
It explicitly types the `obj` to be `Touchables`, and (IMO) `return` is more readable than using a comma here.